### PR TITLE
Cart

### PR DESCRIPTION
**Pull request by surbhi84**
static cart page in react, 
please ignore static data
note that i have decreased browser size a bit so everything fits into one screenshot.

.

### CODE REVIEW

File: src/pages/cart/Cart.jsx [hunk 1]
```diff
@@ -0,0 +1,120 @@
+import cart from "./cart.module.css";
+import prodList from "../productList/productList.module.css";
```
Comment (arpit01923) [inline src/pages/cart/Cart.jsx:2]: use of absolute paths for better readability and understanding

**Review by arpit01923 — COMMENTED** [review]
- use of absolute paths
- use of css part is good

File: src/components/Categories.jsx [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  const [categoryError, setCategoryError] = useState(false);
+
+  useEffect(() => {
+    (async function () {
+      try {
+        let categories = await axios.get("/api/categories");
+        setCategories(categories.data.categories);
+      } catch (err) {
+        setCategoryError(true);
+        console.log(err);
```
Comment (Aditi0505) [inline src/components/Categories.jsx:16]: Please remove console.log

File: src/components/Categories.jsx [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+  }, []);
+  return (
+    <>
+      {nav === "nav" ? (
+        <>
+          {" "}
+          {categoryError && <Error err={"Categories can't be loaded"} />}
+          {categories.map(({ category, link }) => {
+            return (
+              <a
```
Comment (Aditi0505) [inline src/components/Categories.jsx:28]: I think you should use Link instead of <a> tag here.

File: src/components/Categories.jsx [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+              </a>
+            );
+          })}
+        </>
+      ) : (
+        <>
+          {categories.map(({ id, category, link, imgSrc, imgAlt }) => {
+            return (
+              <div className="category-card" key={id}>
+                <a className="text-dec-none" href={link}>
```
Comment (Aditi0505) [inline src/components/Categories.jsx:43]: Same as above.

File: src/pages/cart/cart.module.css [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+
+  .card-ecom-hrz > img {
+    width: 10rem;
+    height: 15rem;
+  }
+
+  .amt-detail {
+    padding: var(--spacing-xs);
+    width: 22rem;
+    /* margin: var(--spacing-xs); */
```
Comment (Aditi0505) [inline src/pages/cart/cart.module.css:59]: Can remove commented code if not needed.

File: src/pages/home/Home.jsx [hunk 6]
```diff
@@ -35,17 +23,17 @@ export const Home = () => {
...
                   <img
-                    src={bCategory.imgSrc}
-                    alt={bCategory.imgAlt}
+                    src={imgSrc}
+                    alt={imgAlt}
                     className="responsive-img opaque"
                   />
-                  <a className="text-overlay-card" href={bCategory.link}>
-                    {bCategory.category}
+                  <a className="text-overlay-card" href={link}>
```
Comment (Aditi0505) [inline src/pages/home/Home.jsx:35]: Can use Link here as well.

**Review by Aditi0505 — APPROVED** [review]
Left few suggestions to look at. Rest looks good to me!

File: src/pages/cart/Cart.jsx (hunk 1 above)
Comment (surbhi84) [reply src/pages/cart/Cart.jsx:2]: if in future i plan to move around my component folder then relative paths won't break while absolute paths will, so i chose to use relative ones and i figure that the second import can be improved, i'll just move out the common css to a common css file.

File: src/components/Categories.jsx (hunk 3 above)
Comment (surbhi84) [reply src/components/Categories.jsx:28]: yeah i will edit that when i start working on features.

**Pull request merged**